pkg/playground/react-patterns.js

```diff
@@ -71,10 +71,11 @@ const onDialogDone = function(success) {
     document.getElementById("demo-dialog-result").textContent = "Dialog closed: " + result + "(" + action + ")";
 };
 
-const onStandardDemoClicked = function(staticError) {
+const onStandardDemoClicked = (staticError) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221201-184951-351895da-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)